tool/automation/framework/app/src/main/kotlin/com/jetbrains/teamcity/Automation.kt

```diff
@@ -25,6 +26,19 @@ class ValidateImage: Subcommand("validate", "Validate Docker Image") {
             throw IllegalArgumentException("Too many image names")
         }
 
+        val images = listOf("jetbrains/teamcity-agent:2022.10-windowsservercore-1809",
```

Comment:
Is the file generated or we need to manually change a version in tags?

Reply:
@burnasheva the list had been created for testing purposes and is not actually used within Kotlin code. It was removed within one of the upstream PRs - https://github.com/JetBrains/teamcity-docker-images/pull/40/files#diff-71f13fe3b08d9f87324d74c4db8ff864408818f79e27982c1323ea8da065210bL29
Please, excuse me for not noticing it earlier.